site/pages/signer/policies/key-concepts-and-examples.mdx

```diff
@@ -5,7 +5,7 @@ description: Key Concepts and Examples of Smart Wallet Policies
 
 # **Key Concepts & Examples**
 
-Smart Wallet Policies are programmatic rules enforced through the offchain policy engine and/or onchain smart contract modules. They govern how a smart wallet interacts with the blockchain, from transaction execution to signer permissions. Unlike traditional embedded wallets, Alchemy Smart Wallets allow you to embed logic that restricts or automates behavior verifiably onchain.
```

Comment:
hmm just curious, why do we remove "offchain" here but add "(onchain or offchain)" in several places below